<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>290</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 170, 0);
font: italic 9pt &quot;Segoe UI&quot;;
</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;chto ischem&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="gde">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>360</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 9pt &quot;Segoe UI&quot;;
color: rgb(217, 255, 201);
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="unicomp">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>80</y>
      <width>80</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 170, 0);
background-color: rgb(29, 35, 23);
font: italic 9pt &quot;Segoe UI&quot;;
</string>
    </property>
    <property name="text">
     <string>C1+C2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="chto">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>310</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 9pt &quot;Segoe UI&quot;;
color: rgb(217, 255, 201);
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="unibag">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>220</y>
      <width>80</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 170, 0);
background-color: rgb(29, 35, 23);
font: italic 9pt &quot;Segoe UI&quot;;
</string>
    </property>
    <property name="text">
     <string>B1+B2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="incomp">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>80</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 170, 0);
background-color: rgb(29, 35, 23);
font: italic 9pt &quot;Segoe UI&quot;;
</string>
    </property>
    <property name="text">
     <string>in compi</string>
    </property>
   </widget>
   <widget class="QPushButton" name="search">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>390</y>
      <width>80</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 170, 0);
background-color: rgb(29, 35, 23);
font: italic 9pt &quot;Segoe UI&quot;;
</string>
    </property>
    <property name="text">
     <string>search</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>450</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 170, 0);
font: italic 9pt &quot;Segoe UI&quot;;
</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;result&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="inbag">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>150</y>
      <width>80</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 170, 0);
background-color: rgb(29, 35, 23);
font: italic 9pt &quot;Segoe UI&quot;;
</string>
    </property>
    <property name="text">
     <string>in bagi</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>340</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 170, 0);
font: italic 9pt &quot;Segoe UI&quot;;
</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;gde ischem&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTableView" name="incomp1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>311</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 9pt &quot;Segoe UI&quot;;
color: rgb(217, 255, 201);
</string>
    </property>
   </widget>
   <widget class="QTableView" name="incomp2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>311</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 9pt &quot;Segoe UI&quot;;
color: rgb(217, 255, 201);
</string>
    </property>
   </widget>
   <widget class="QTableView" name="outcomp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>311</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 9pt &quot;Segoe UI&quot;;
color: rgb(217, 255, 201);
</string>
    </property>
   </widget>
   <widget class="QTableView" name="outbag">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>390</y>
      <width>321</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 9pt &quot;Segoe UI&quot;;
color: rgb(217, 255, 201);
</string>
    </property>
   </widget>
   <widget class="QTableView" name="inbag1">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>321</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 9pt &quot;Segoe UI&quot;;
color: rgb(217, 255, 201);
</string>
    </property>
   </widget>
   <widget class="QTableView" name="inbag2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>210</y>
      <width>321</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 9pt &quot;Segoe UI&quot;;
color: rgb(217, 255, 201);
</string>
    </property>
   </widget>
   <widget class="QTableView" name="ressearch">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>470</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 9pt &quot;Segoe UI&quot;;
color: rgb(217, 255, 201);
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 170, 0);
font: italic 9pt &quot;Segoe UI&quot;;
</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Comp1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>190</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 170, 0);
font: italic 9pt &quot;Segoe UI&quot;;
</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Comp2&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>370</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 170, 0);
font: italic 9pt &quot;Segoe UI&quot;;
</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Comp&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 170, 0);
font: italic 9pt &quot;Segoe UI&quot;;
</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Bag1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>190</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 170, 0);
font: italic 9pt &quot;Segoe UI&quot;;
</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Bag2&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>370</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 170, 0);
font: italic 9pt &quot;Segoe UI&quot;;
</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Bag&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
